qt/BaseDialog.h

```diff
@@ -14,6 +14,7 @@
 
 class BaseDialog: public QDialog
 {
+  Q_OBJECT
```

Comment:
`Q_OBJECT` neither required nor needed here, as this class doesn't use Qt's metasystem.